pages/common/transcrypt.md

```diff
@@ -0,0 +1,24 @@
+# transcrypt
+
+> Transparently encrypt files within a git repository.
+> More information: <https://github.com/elasticdog/transcrypt>.
+
+- Initialize an unconfigured repository:
+
+`transcrypt`
+
+- List the currently encrypted files:
+
+`git ls-crypt`
+
+- Display the credentials of a configured repository:
+
+`transcrypt --display`
+
+- Initialize and decrypt a fresh clone of a configured repository:
+
+`transcrypt --cipher=<cipher>`
```

Comment:
Perhaps we could correct this to use the `{{token_syntax}}`?
```suggestion
`transcrypt --cipher={{cipher}}`
```

Reply:
Definitely yes, commiting your suggestion :)